waspc/examples/todoApp/.env.server.example

```diff
@@ -0,0 +1,20 @@
+TEST_ENV_VAR="I am test"
+
+# Dummy values here will allow app to run, but you will need real values to get Google Auth to work.
```

Comment:
Hmm, seems a little hacky. Why do we need this? Won't dotenv set the env vars?

Reply:
I explain this more in the README.md .

So yes, vault-dotenv will pull in .env.server for us. And therefore, we don't need .env.server.example. But, what if you don't have access to .env.server? And external contributors don't. Then, they can still copy .env.server.example into .env.server and get sometihng running thanks to these dummy values. .env.server.example also serves as a documentation for them, to let them know which env vars we expect and for what. This is how we also do it for open-saas and for other templates.